spec/services/document_capture_session_result_spec.rb

```diff
@@ -47,5 +51,66 @@
         expect(result.selfie_status).to be_an_instance_of(Symbol)
       end
     end
+
+    describe '#success?' do
+      it 'reports true when doc_auth_success is true and selfie_status is :not_processed' do
+        result = DocumentCaptureSessionResult.new(
+          id: id,
+          success: false,
+          pii: pii,
+          attention_with_barcode: false,
+          selfie_status: :not_processed,
+          doc_auth_success: true,
+        )
+        expect(result.success?).to eq(true)
+      end
+      it 'reports failure when selfie_status is :fail' do
+        result = DocumentCaptureSessionResult.new(
+          id: id,
+          success: false,
+          pii: pii,
+          attention_with_barcode: false,
+          selfie_status: :fail,
+          doc_auth_success: true,
+        )
+        expect(result.success?).to eq(false)
+      end
+
+      it 'reports failure when doc_auth_success is false' do
+        result = DocumentCaptureSessionResult.new(
+          id: id,
+          success: false,
+          pii: pii,
+          attention_with_barcode: false,
+          selfie_status: :success,
+          doc_auth_success: false,
+        )
+        expect(result.success?).to eq(false)
+      end
+
+      describe 'hypothetically when old success field and contradicting new status filed' do
```

Comment:
This is a complete nit, but I think it might be clearer to specify the fields.....  Something like
```suggestion
      describe 'when success field, doc_auth_success, and selfie_status conflict' do
```